ai21/models/assistant/run.py

```diff
@@ -0,0 +1,41 @@
+from typing import Literal, Any, List
+
+from typing_extensions import TypedDict
+
+
+RunStatus = Literal[
+    "cancelled",
+    "cancelling",
+    "completed",
+    "expired",
+    "failed",
+    "incomplete",
+    "in_progress",
+    "queued",
+    "requires_action",
+]
+
+
+class ToolOutput(TypedDict):
+    tool_call_id: str
+    output: Any
+
+
+class Function(TypedDict):
+    name: str
+    arguments: Any
```

Comment:
Is this really any? is it not List/Dict or anything? can it also be a single `int`?

Reply:
yeah :(